crates/biome_cli/tests/snapshots/main_commands_init/creates_config_file.snap

```diff
@@ -26,22 +26,28 @@ Welcome to Biome! Let's get you started...
 
 Files created ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 
-  - biome.json: Your project configuration. Documentation: https://biomejs.dev/reference/configuration
+  - biome.json
+    Your project configuration. See https://biomejs.dev/reference/configuration
 
 Next Steps ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 
   1. Setup an editor extension
-     Get live errors as you type and format when you save. Learn more: https://biomejs.dev/guides/getting-started#editor-setup
+     Get live errors as you type and format when you save.
+     Learn more: https://biomejs.dev/guides/integrate-in-editor/
 
   2. Try a command
-     biome ci checks for lint errors and verifies formatting. Run biome --help for a full list of commands and options.
+     biome check verifies formatting and lints.
```

Comment:
My suggestion: verifies that formatting, import statements are sorted and lint rules are respected.